crates/rover-client/src/error.rs

```diff
@@ -4,7 +4,7 @@ use thiserror::Error;
 #[derive(Error, Debug)]
 pub enum RoverClientError {
     /// The provided GraphQL was invalid.
-    #[error("encountered a GraphQL error, registry responded with: {msg}")]
+    #[error("GraphQL error: {msg}")]
```

Comment:
I wonder if we should move the message to the next line and give it some 'breathing room' so it's a bit more readable:
```suggestion
    #[error("GraphQL error: \n\n{msg}")]
```